app/services/retrieve_position_data.rb

```diff
@@ -47,10 +47,15 @@ def query_format
         OPTIONAL { ?position pq:P768 ?district . }
         OPTIONAL { ?position pqv:P580 [wikibase:timeValue ?position_start; wikibase:timePrecision ?position_start_precision] . }
         OPTIONAL { ?merged_then_deleted owl:sameAs ?person }
+        BIND(ceil((year(?term_start) - year(?position_start)) * 365.2425 +
+                  (month(?term_start) - month(?position_start)) * 30.4375 +
+                  (day(?term_start) - day(?position_start))) AS ?days_between)
         FILTER (!bound(?term) || ?term = ?page_term)
         FILTER (
           !bound(?term_start) || !bound(?position_start) ||
-          ?term_start <= ?position_start
+          (?position_start_precision = 9 && year(?term_start) = year(?position_start)) ||
+          (?position_start_precision = 10 && year(?term_start) = year(?position_start) && month(?term_start) = month(?position_start)) ||
+          (?position_start_precision = 11 && 0 <= ?days_between && ?days_between < 28)
```

Comment:
I might be missing something here, but shouldn't we drop the `0 <= ?days_between` part of the condition? If someone's position started a few days into the term, we wouldn't want to create a new duplicate statement for them (or we'd at least want to know about that existing P39).

Reply:
Been struggling to get my head around this but I think you're correct and have removed that part of the conditional